doc/sphinx/language/core/assumptions.rst

```diff
@@ -202,10 +202,12 @@ has type :n:`@type`.
    :name: ‘ident’ already exists. (Axiom)
    :undocumented:
 
-.. warn:: @ident is declared as a local axiom
+.. warn:: Use of "Variable" or "Hypothesis" outside sections behaves as "#[local] Parameter" or "#[local] Axiom".
```

Comment:
```suggestion
.. error:: Use of "Variable" or "Hypothesis" outside sections behaves like "#[local] Parameter" or "#[local] Axiom".
```

If it's an error by default, let's show it in the manual as an error.

Reply:
Error would probably be `.. exn::` but this is still technically a warning. The fact that it's an error by default doesn't prevent userzs from disabling or silencing it like any other warning. I think we tend to keep documenting those are warnings then.